<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_pages">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>771</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page_1_home">
        <widget class="QLabel" name="label_page_1_home">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>95</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>page 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_borrowedBooks_2">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>190</y>
           <width>200</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Borrowed Books</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_allBooks_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>200</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>All Books</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_availableBooks_2">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>190</y>
           <width>200</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Available Books</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2_members">
        <widget class="QLabel" name="label_page_2_members">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>171</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Members</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_search_members">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>200</y>
           <width>650</width>
           <height>29</height>
          </rect>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_members">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>700</width>
           <height>624</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>890</y>
           <width>314</width>
           <height>29</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="pushButton_borrow_book">
          <property name="text">
           <string>Borrow</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_update">
          <property name="text">
           <string>refresh</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit"/>
        </widget>
        <widget class="QPushButton" name="pushButton_delete_member">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>890</y>
           <width>96</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_search_members">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>50</width>
           <height>29</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3_books">
        <widget class="QLabel" name="label_page_3_books">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>141</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>All Books</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_borrowedBooks">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>60</y>
           <width>200</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Borrowed Books</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_allBooks">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>200</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>All Books</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_availableBooks">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>60</y>
           <width>200</width>
           <height>100</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Available Books</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_books">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>700</width>
           <height>624</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_search_book">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>50</width>
           <height>29</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_search_books">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>200</y>
           <width>650</width>
           <height>29</height>
          </rect>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_borrow_book_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>890</y>
           <width>85</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Borrow</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="label_page_4_borrowed_books">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>241</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Borrowed Books</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_borrowed_books">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>700</width>
           <height>624</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_search_borrowed_books">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>200</y>
           <width>650</width>
           <height>29</height>
          </rect>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_search_borrowed_books">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>50</width>
           <height>29</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_pages_navigator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>109</y>
      <width>100</width>
      <height>891</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton_page1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_page2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Members</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_page3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>All Books</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_page4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Available Books</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_page5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrowed Books</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_library_icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Libraray</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
